sdk/storage/azure-storage-blob-cryptography/README.md

```diff
@@ -21,7 +21,7 @@ This package supports client side encryption for blob storage.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.6.2</version>
+  <version>12.7.0</version>
```

Comment:
why do we bump version ?

Reply:
We forgot to bump the version with the last release. This is bringing it up to latest.